relayer/src/link/relay_path.rs

```diff
@@ -390,17 +393,24 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> RelayPath<ChainA, ChainB> {
             .map_err(|e| LinkError::query(self.src_chain().id(), e))?;
         let dst_latest_height = dst_latest_info.height;
         // Operational data targeting the source chain (e.g., Timeout packets)
-        let mut src_od = OperationalData::new(dst_latest_height, OperationalDataTarget::Source);
+        let mut src_od = OperationalData::new(
+            dst_latest_height,
+            OperationalDataTarget::Source,
+            events.tracking_id(),
+        );
         // Operational data targeting the destination chain (e.g., SendPacket messages)
-        let mut dst_od = OperationalData::new(src_height, OperationalDataTarget::Destination);
+        let mut dst_od = OperationalData::new(
+            src_height,
+            OperationalDataTarget::Destination,
+            events.tracking_id(),
+        );
 
         for event in input {
-            debug!("[{}] {} => {}", self, self.src_chain().id(), event);
+            trace!("[{}] {} => {}", self, self.src_chain().id(), event);
```

Comment:
This could use more KV pairs for legibility. The span already gives the tracking ID, so the `self` parameter replicates it (and adds more non-KV clutter)?

Reply:
This is how this log line looks in practice:

> 2021-12-22T12:18:40.052114Z TRACE ThreadId(443) generate{id=BS6CBKeio4}: [ibc-0:transfer/channel-0 -> ibc-1] ibc-0 => SendPacketEv(SendPacket - h:0-25087, seq:9, path:channel-0/transfer->channel-0/transfer, toh:0-0, tos:Timestamp(2021-12-22T12:21:29.002094Z)))


I think `self` does not replicate the tracking ID, but displays the relaying path: `ibc-0:transfer/channel-0 -> ibc-1` in this case.

What would be a concrete suggestion to replace this log using KV syntax?

Reply:
With the span rework, most of this information should be redundant.